src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { Person } from '../types';
+import PersonLink from './PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+const PeopleTable: React.FC<Props> = ({ people }) => {
+  const showPeople = people?.map(person => {
+    return <PersonLink person={person} people={people} key={person.slug} />;
```

Comment:
The `PersonLink` component is being passed a `people` prop, but it does not seem to be used within `PersonLink`. If the prop is not used, it should be removed to prevent passing unnecessary data.